docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<p><span class="mention" data-value="Eyal-Danieli" data-denotation-char="@">﻿<span contenteditable="false">@Eyal-Danieli</span>﻿</span> this example creates an alerts for all the model endpoints result right ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
@Eyal-Danieli - pls comment